src/modules/CreateChannel/components/CreateChannelUI/create-channel-ui.scss

```diff
@@ -29,7 +29,6 @@
   @include mobile() {
     min-width: 288px;
     width: 100%;
-    padding-right: 0;
```

Comment:
The channel creation dialog layout won't be broken even without this. 

#### LTR
![Screenshot 2024-06-13 at 4 36 26 PM](https://github.com/sendbird/sendbird-uikit-react/assets/10060731/dc60968c-cd97-4ff7-b324-c402a959368c)

#### RTL
![Screenshot 2024-06-13 at 4 36 22 PM](https://github.com/sendbird/sendbird-uikit-react/assets/10060731/38f3e065-be10-4163-80a7-5c0e4366789a)